### PR TITLE
Switch gpt-j-6b model for gpt2-xl

### PR DESCRIPTION
**Pull request by freddyaboulton**
## Description

The demos using gpt-j-6b were not launching because they rely on the inference api and the inference api no longer supports this model. Switching to https://huggingface.co/gpt2-xl instead. I renamed the demos as well for consistency and searched for mentions of gpt-j in the guides and docstrings.

Demo built here with new model:
https://huggingface.co/spaces/gradio/gpt_j
https://huggingface.co/spaces/gradio/gpt_j_unified/

## 🎯 PRs Should Target Issues

Before your create a PR, please check to see if there is [an existing issue](https://github.com/gradio-app/gradio/issues) for this change. If not, please create an issue before you create this PR, unless the fix is very small. 

Not adhering to this guideline will result in the PR being closed. 

## Tests & Changelog

1. PRs will only be merged if tests pass on CI. To run the tests locally, please set up [your Gradio environment locally](https://github.com/gradio-app/gradio/blob/main/CONTRIBUTING.md) and run the tests: `bash scripts/run_all_tests.sh`

1. You may need to run the linters: `bash scripts/format_backend.sh` and `bash scripts/format_frontend.sh`
  
1. Unless the pull request is labeled with the "no-changelog-update" label by a maintainer of the repo, all pull requests must update the changelog located in `CHANGELOG.md`:

Please add a brief summary of the change to the Upcoming Release section of the `CHANGELOG.md` file and include
a link to the PR (formatted in markdown) and a link to your github profile (if you like). For example, "* Added a cool new feature by `[@myusername](link-to-your-github-profile)` in `[PR 11111](https://github.com/gradio-app/gradio/pull/11111)`".

### CODE REVIEW

File: demo/stt_or_tts/run.py [hunk 1]
```diff
@@ -1,7 +1,5 @@
 import gradio as gr
 
-title = "GPT-J-6B"
```
Comment (freddyaboulton) [inline demo/stt_or_tts/run.py:3]: This title was not being used

**Comment by abidlabs** [comment]
Thanks @freddyaboulton for fixing these demos. There's also a couple more I noticed that use `gpt-j`: `autocomplete` and `blocks_gpt`. I can go ahead and push fixes for these

**Comment by abidlabs** [comment]
Pushed the changes -- will go ahead and merge after CI so that our demos are back up

**Comment by freddyaboulton** [comment]
Thanks for the fix @abidlabs !!

**Pull request merged**